scripts/xtensa-build-all.sh

```diff
@@ -218,6 +224,12 @@ do
 		HOST="xtensa-cnl-elf"
 		XTENSA_TOOLS_VERSION="RF-2016.4-linux"
 	fi
+	# override default ARCH if BUILD_FORCE_UP is set
+	if [ $BUILD_FORCE_UP == yes ]
```

Comment:
maybe `if [ "$BUILD_FORCE_UP" == "yes" ]` is safer in bash.

Reply:
OK, thanks, updated.